### ISSUE TITLE
remove "TODO" from items in widget TODO view.

### ISSUE THREAD

**Issue by naught101** [orgzly/orgzly-android#320]
It would be good if there was a way to remove the "TODO" prefix on todos in the widget, because they block most of the text, and make the actual list unreadable, especially when the widget is relatively narrow. I'll add a screenshot.

**Comment by naught101** [comment]
![phimpme1524804958707](https://user-images.githubusercontent.com/167164/39345480-fbbf81fe-4a2b-11e8-9019-0c68af687913.png)

**Comment by NiceFeather** [comment]
Acturally all of widget can be resize in the most of home lanucher, long press&resizing it will be OK.

**Comment by naught101** [comment]
Yes, I know, but I would prefer for the widget to not take up the wholes screen. For example, my current layout looks like this: 

![screenshot_20180427-145432](https://user-images.githubusercontent.com/167164/39384968-e28fee18-4ab1-11e8-9294-5fdf3c5b863c.png)

I would like to keep the TODO list on my main home screen, but if it is any bigger than this, then it takes up too much space and my home screen becomes less useful as a whole.

**Comment by NiceFeather** [comment]
Yes, @nevenz maybe set an option for this state to show/hide is more flexible like 'Inherited tags in search results' of tag, this is just a proposal.

BTW, the option of 'Inherited tags in search results' above seems has a bug, all note tags always show up.

![screenshot_20180428-114811](https://user-images.githubusercontent.com/28109556/39393521-b946719a-4ab5-11e8-984f-33de62ccfd03.jpg)

**Comment by nevenz** [comment]
Would removing checkmark be enough? Note would be selectable instead and action menu would open. I'm not a fan of allowing state to be removed as it's pretty important part of the note. But if we must, I guess it could be allowed for the widget only (as a part of #142).

> BTW, the option of 'Inherited tags in search results' above seems has a bug, all note tags always show up.

Note's yes, but its ancestors' shouldn't.

With option off:

![off](https://user-images.githubusercontent.com/10671476/39430621-974d795a-4c8e-11e8-9825-6c7d50a64499.png)

With option on:

![on](https://user-images.githubusercontent.com/10671476/39430680-c189716a-4c8e-11e8-8a5a-a06ecf87df3c.png)

**Comment by naught101** [comment]
Re: state removal: I would only suggest that for a state-based search, I guess. Everything having the same state makes the state information redundant.

Another option might be to replace "TODO" with a small logo, or just a small "T".

Yes, I would also appreciate the check mark being removable. I don't really use them much (just delete done things, mostly).

I guess the margins could be narrowed a bit too (see the Etar widget in the screenshot for example), and it would be nice to have the option to se the widget font size smaller.

**Comment by Sirove** [comment]
+1!

**Comment by lolilolicon** [comment]
> Another option might be to replace "TODO" with a small logo, or just a small "T".

May I suggest a simple workaround?

You can change in the preferences the "keywords for active (todo) tasks" to emojis.
From TODO NEXT to 🔜 🔛 for example.
Or use an ideograph or something else, if they make sense to you.

**Comment by nightscape** [comment]
@lolilolicon I like your suggestion, but if implemented like that it could break interop with other org implementations or at least make it hard to type this on a PC.
How about we allow the same syntax that org-mode allows for creating shortcuts in the state settings and provide an option to show the shortcut in the widget?
Something like
```
TODO(T) NEXT(N)

Or

TODO(🔜) NEXT(🔛)
```
That way one could keep the text based state and the user could choose how it should look like in the widget (and possibly other places?)

**Comment by torwag** [comment]
Maybe some save spacing icon / symbol instead of the text would be a solution... A red dot for TODO, orange for in progress, green for done, etc.
